Sources/DOT/DOTRenderer.swift

```diff
@@ -29,10 +29,16 @@ fileprivate func which(_ command: String) throws -> URL {
     return URL(fileURLWithPath: string)
 }
 
-extension Graph {
-    public func render(using layout: LayoutAlgorithm, to format: Format) throws -> Data {
-        let encoded = DOTEncoder().encode(self)
+public struct DOTRenderer {
+    private let layout: LayoutAlgorithm
```

Comment:
```suggestion
    let layout: LayoutAlgorithm
```

No harm in letting API consumers be able to tell which layout a renderer uses. I could even see a case for making this a `var`.